transports/msmq/index.md

```diff
@@ -71,12 +71,14 @@ Although MSMQ has the concept of both [Public and Private queues](https://techne
 
 ## Permissions
 
-| Group          | Permissions          |
-|----------------|----------------------|
-| Owning account | Write, Receive, Peek |
-| Administrators | Full                 |
-| Anonymous      | Write                |
-| Everyone       | Write                |
+| Group          | Permissions         | Description                       |
+|----------------|---------------------|-----------------------------------|
+| Owning account | Send, Receive, Peek | Set by NServiceBus                |
+| Administrators | Full                | Set by NServiceBus                |
+| Anonymous      | Send                | Set by NServiceBus prior to 6.1.0 |
```

Comment:
@ramonsmits - Shouldn't this be Versions 6.0.10 and below instead of 6.1.0? 6.1.0 was the version that introduced tighter permissions?